src/Files.Uwp/Filesystem/Drives.cs

```diff
@@ -98,40 +96,37 @@ private void StartDeviceWatcher()
             }
             else
             {
-                DeviceWatcher_EnumerationCompleted(null, null);
+                DeviceWatcher_EnumerationCompleted();
             }
         }
 
-        private async void DeviceWatcher_EnumerationCompleted(DeviceWatcher sender, object args)
+        private void DeviceWatcher_EnumerationCompleted(DeviceWatcher sender = null, object args = null)
         {
             System.Diagnostics.Debug.WriteLine("DeviceWatcher_EnumerationCompleted");
-            await RefreshUI();
+            RefreshUI();
             folderSizeProvider.CleanCache();
         }
 
-        private async Task RefreshUI()
+        private async void RefreshUI(CoreApplicationView view = null, IActivatedEventArgs args = null)
```

Comment:
Why remove Task?

Reply:
A considerable amount of the methods in `DrivesManager` double as event handlers. In the meantime, I made `RefreshUI` only act as an event handler, and we should consider the same thing for `DeviceWatcher_EnumerationCompleted`. Except this might be unnecessary once we decouple DrivesManager from the UI.